Rocky.class.nut

```diff
@@ -53,7 +83,23 @@ class Rocky {
     // Requests
     function on(verb, signature, callback, timeout=null) {
         //Check timeout and set it to class-level timeout if not specified for route
-        if (timeout == null) timeout = this._timeout;
+        if (timeout == null) {
+            timeout = this._timeout;
+        }
+
+        // Validate paramters
+        if (!(typeof verb == "string")) {
```

Comment:
again, I don't really think it should be here.

Reply:
Same here, happy to remove.